qiskit/circuit/library/data_preparation/zz_feature_map.py

```diff
@@ -33,7 +33,7 @@ class ZZFeatureMap(PauliFeatureMap):
         └───┘└─────────────────┘                                  └───┘└──────────────────────┘└───┘
 
     where ``φ`` is a classical non-linear function, which defaults to ``φ(x) = x`` if and
-    ``φ(x,y) = (pi - x)(pi - y)``.
+    ``φ(x,y) = (\pi - x)(\pi - y)``.
```

Comment:
This is actually just in a "code" role not a LaTeX one, so there's probably no need to use LaTeX macros.  That said, if you wanted to switch the things that are logically inline equations here to be rendered with `` :math:`\varphi(x) = x` `` and so on, that would be welcome.

Reply:
Thanks for the review. I added the `:math:` .